src/diagnosebasic.cpp

```diff
@@ -55,6 +55,8 @@
 
 using namespace MOBase;
 
+const QRegularExpression DiagnoseBasic::RE_LOG_FILE(".*log[0-9]*$");
```

Comment:
This is not strictly equivalent, it doesn't check for a dot before the extension. Not sure if it's a problem.

Reply:
I've added the dot back.